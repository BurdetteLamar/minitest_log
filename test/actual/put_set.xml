<log>
  <data_ name='put_set' class='Set' method=':each_with_index' size='3'>
    <![CDATA[
0: a
1: aa
2: aaa
]]>
  </data_>
</log>
