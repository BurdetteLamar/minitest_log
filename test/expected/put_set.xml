<log>
  <each_with_index_ name='put_set' class='Set' size='3'>
    <![CDATA[
0: a
1: aa
2: aaa
]]>
  </each_with_index_>
</log>
